persistqueue/__init__.py

```diff
@@ -4,10 +4,13 @@
 __version__ = '0.4.1'
 
 from .exceptions import Empty, Full  # noqa
-from .pdict import PDict  # noqa
 from .queue import Queue  # noqa
-from .sqlqueue import SQLiteQueue, FIFOSQLiteQueue, FILOSQLiteQueue, UniqueQ  # noqa
-from .sqlackqueue import SQLiteAckQueue
+try:
+    from .pdict import PDict  # noqa
+    from .sqlqueue import SQLiteQueue, FIFOSQLiteQueue, FILOSQLiteQueue, UniqueQ  # noqa
+    from .sqlackqueue import SQLiteAckQueue
+except ImportError as error:
+    pass
```

Comment:
Can you please log a message for user: `no sqlite3 module found, sqlite3 based queues are not available`?